Functions.Templates/Templates/SqlInputBinding-CSharp/.template.config/template.json

```diff
@@ -36,21 +41,22 @@
     "defaultName": "SqlInputBindingCSharp",
     "postActions": [
         {
-          "Description": "Adding Reference to Microsoft.Azure.WebJobs.Extensions.Sql Nuget package",
-          "ActionId": "B17581D1-C5C9-4489-8F0A-004BE667B814",
-          "ContinueOnError": "true",
-          "ManualInstructions": [],
-          "args": {
-            "referenceType": "package",
-            "reference": "Microsoft.Azure.WebJobs.Extensions.Sql", "version": "0.1.457-preview",
```

Comment:
This is kept in a single line on purpose, for easy search and replacement of extension versions